test/index.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable global-require */
+const testFolder = './test';
+const fs = require('fs');
+
+fs.readdir(testFolder, (err, files) => {
+  files.forEach((file) => {
+    if (file !== 'testRoutes.js') {
+      // eslint-disable-next-line import/no-dynamic-require
+      require(`./${file}`);
+      console.log(file);
```

Comment:
delete the console.log 
and you should handle the err for the fs.readdir(testFolder, (err, files) =>